### PR TITLE
Clean notebooks

### PR DESCRIPTION
**Pull request by jennyfolkesson**
I've added a notebooks folder where I put two notebooks (one that was already in the repo, one from the DL workshop). I've stripped outputs for a cleaner commit history.

I also changed the train script so you can call the function directly, bypassing the cli.

@pranathivemuri I've added you as a reviewer. You don't have to review if you don't have time, it's just if you want to start getting familiar with the repo.

### CODE REVIEW

File: conda_environment.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - pydot
+  - PyYAML
+  - tensorflow-gpu
+  - testfixtures
+  - scikit-image
+  - scikit-learn
+  - scipy
+  - pip
+  - pip:
+    - keras==2.1.6
```
Comment (anithapk) [inline conda_environment.yml]: 👍

File: micro_dl/cli/image_inference.py [hunk 2]
```diff
@@ -94,7 +94,15 @@ def parse_args():
     return args
 
 
-def run_prediction(args, gpu_ids, gpu_mem_frac):
+def run_prediction(model_dir,
```
Comment (anithapk) [inline micro_dl/cli/image_inference.py]: I'm so sorry, I moved this file to deprecated in the previous PR. We can keep this in cli, let's remove it from deprecated!

File: notebooks/segmentation_masks.ipynb [hunk 3]
```diff
@@ -0,0 +1,158 @@
+{
```
Comment (anithapk) [inline notebooks/segmentation_masks.ipynb]: Sorry again. moved gen_masks.py to deprecated in the previous PR. We could delete this?

File: micro_dl/cli/image_inference.py (hunk 2 above)
Comment (jennyfolkesson) [reply micro_dl/cli/image_inference.py]: How about we keep it in cli until I've tested the inference module with data other than 3D and updated the notebook, then I can move it to deprecated.

File: notebooks/segmentation_masks.ipynb (hunk 3 above)
Comment (jennyfolkesson) [reply notebooks/segmentation_masks.ipynb]: Yes! It was just a notebook I found in the repo, but if nobody's using it let's delete it.

File: micro_dl/cli/image_inference.py (hunk 2 above)
Comment (anithapk) [reply micro_dl/cli/image_inference.py]: sorry what I meant was move it from deprecated to cli!

**Pull request merged**